src/backend/agent.js

```diff
@@ -492,7 +494,9 @@ export default class Agent extends EventEmitter {
     window.removeEventListener('click', this._onClick, true);
```

Comment:
Should we still be using "click" for the signal to stop inspecting? Or should we move that to pointer down as well?

Reply:
Ah, I guess we can't cleanup in pointer up or we wouldn't block the "click" huh.

Reply:
Yep, we need to keep "click".